### ISSUE TITLE
[Audio Player]: QA: Focus resetting to the start after selecting option in 'Playback speed rate'/'Closed Caption'

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9283]
### Description

Page Element: After selecting any option in 'Playback speed rate'/'Closed Caption'

Issue Description:
Focus resetting to the start after selecting option in 'Playback speed rate'/'Closed Caption'

Expected:
Focus should stay in the same respective element after selecting option in 'Playback speed rate'/'Closed Caption'

User Impact:
When standard methods of keyboard navigation is not supported, this impact the user ability to interact with the page controls. This impacts Keyboard only users/Screen reader users

### Component(s) impacted

Audio Player

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
